web/src/portico/team.ts

```diff
@@ -1,9 +1,31 @@
 import $ from "jquery";
 import _ from "lodash";
 
-import {page_params} from "../page_params";
+import * as blueslip from "../blueslip";
+
+const all_repository_names = [
+    "zulip",
+    "zulip-desktop",
+    "zulip-mobile",
+    "python-zulip-api",
+    "zulip-js",
+    "zulipbot",
+    "zulip-terminal",
+    "zulip-ios-legacy",
+    "zulip-android",
+] as const;
+type RepositoryNameType = (typeof all_repository_names)[number];
```

Comment:
This is just `RepositoryName`. A type should not be named with a `Type` suffix unless it’s the type *of a type*. (Otherwise we would have `StringType`, `NumberType`, etc.)